vnext/Desktop/WebSocketModule.cpp

```diff
@@ -15,16 +15,21 @@ using namespace folly;
 
 using std::string;
 
-namespace Microsoft {
-namespace React {
+namespace {
+
+static string s_moduleName = "WebSocketModule";
```

Comment:
>static string [](start = 0, length = 13)

Global static std::strings have perf implications .. as they gets constructed while loading the dll .. Could you change this to "const char*"

Reply:
Wouldn't this mean creating a new `std::string` based on this `const char*` each time `getName()` is invoked?